### PR TITLE
[PythonFilter/Testcases] V2: Add some testcases for Python filter

### PR DESCRIPTION
**Pull request by dongju-chae**
I've added some existing testcases
- passthrough.py
- scaler.py

Changelog
V2: update codes to check the python version
V1: add two testcases

Signed-off-by: Dongju Chae <dongju.chae@samsung.com>

### CODE REVIEW

**Comment by myungjoo** [comment]
Wow.. these "python custom neural-networks" appear so easy to implement! Good!

File: ext/nnstreamer/tensor_filter/tensor_filter_python_core.cc [hunk 1]
```diff
@@ -47,11 +47,15 @@ PYCore::PYCore (const char* _script_path, const char* _custom)
    * (e.g., multiarray.x86_64-linux-gnu.so: undefined symbol: PyExc_SystemError)
    */
   gchar libname[32];
-#if PY_VERSION_HEX >= 0x03000000
-  g_snprintf (libname, sizeof(libname), "libpython%fm.so.1", PYTHON3_VERSION);
+
+  g_snprintf (libname, sizeof(libname), 
+#if PY_VERSION_HEX >= 0x03000000
```
[the remark this answers is not in the document]
Comment (dongju-chae) [reply ext/nnstreamer/tensor_filter/tensor_filter_python_core.cc]: Yes. I had to fix this as the previous one has some bugs to find the lib name.

File: tests/nnstreamer_filter_python/checkScaledTensor.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+#         This script is imported from tests/nnstreamer_filter_custom/
+# @author MyungJoo Ham <myungjoo.ham@samsung.com>
+
+import sys
+
+## @brief Compare original content and scaled one
+def compare (data1, width1, height1, data2, width2, height2, innerdim):
+  if (len(data1) * width2 * height2) != (len(data2) * width1 * height1):
+    print(str(len(data1) * width2 * height2)+" / "+str(len(data2) * width1 * height1))
+    return 1
```
[the remark this answers is not in the document]
Comment (dongju-chae) [reply tests/nnstreamer_filter_python/checkScaledTensor.py]: In fact, checkScaledTensor.py is not my code. I reused this script in tests/nnstreamer_filter_custom/.
I think the return values will be used in runTest.sh.

[the remark this answers is not in the document]
Comment (wooksong) [reply tests/nnstreamer_filter_python/checkScaledTensor.py]: @dongju-chae I see.

**Pull request merged**